spec/factories/responses.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :response do
+    response ""
+    user nil
```

Comment:
Why is the user nil and response empty? Maybe you could try use a Faker for this